libs/service-portal/settings/access-control/src/components/AccessItem/AccessItem.tsx

```diff
@@ -0,0 +1,199 @@
+              <GridColumn
+                span={['12/12', '12/12', '3/12']}
+                className={styles.item}
+              >
+                <Box
+                  paddingBottom={isLastItem ? 'p3' : 'p1'}
+                  paddingTop={isFirstItem ? 'p3' : 'p1'}
+                  paddingLeft={isFirstItem ? 0 : [0, 0, 4]}
+                  display="flex"
+                  alignItems="center"
+                >
+                  <CheckboxController
+                    id={`${item.model}.name`}
+                    spacing={0}
+                    labelVariant={isFirstItem ? 'default' : 'medium'}
+                    defaultValue={existingScope ? [existingScope.name] : []}
+                    options={[
+                      {
+                        label: item.displayName,
+                        value: item.name,
+                      },
+                    ]}
+                    onSelect={(value) => onSelect(item, value)}
+                  />
+                </Box>
+              </GridColumn>
+              <GridColumn
+                span={['12/12', '12/12', '4/12', '5/12']}
+                className={styles.item}
+              >
+                <Box
+                  paddingBottom={isLastItem ? 'p3' : 'p1'}
+                  paddingTop={isFirstItem ? 'p3' : 'p1'}
+                  paddingLeft={[0, 0, 3]}
+                >
+                  <Text variant={isFirstItem ? 'default' : 'medium'}>
+                    {item.description}
+                  </Text>
+                </Box>
+              </GridColumn>
+              <GridColumn span={['8/12', '8/12', '5/12', '4/12']}>
+                <Box
+                  paddingBottom={isLastItem ? 'p3' : 'p1'}
+                  paddingTop={isFirstItem ? 'p3' : 'p1'}
+                  paddingLeft={[0, 0, 3]}
+                >
+                  <div className={cs(isSelected ? undefined : styles.hidden)}>
+                    <DatePickerController
+                      id={`${item.model}.validTo`}
+                      size="sm"
+                      label={formatMessage({
+                        id:
+                          'sp.settings-access-control:access-item-datepicker-label',
+                        defaultMessage: 'Dagsetning til',
+                      })}
+                      backgroundColor="blue"
+                      minDate={new Date()}
+                      defaultValue={
+                        existingScope?.name
+                          ? existingScope.validTo
+                          : format(defaultDate, 'yyyy-MM-dd')
+                      }
+                      locale={lang}
+                      placeholder={undefined}
+                      onChange={(value) => onChange(item, value)}
+                    />
+                  </div>
+                </Box>
+              </GridColumn>
+            </GridRow>
+            <Box paddingY={1}>
+              <Divider />
+            </Box>
```

Comment:
IMHO the vertical spacing on mobile is a bit too tight around the divider and a bit too airy around the description. What do you think?

Reply:
Agree - fixed ✅